### ISSUE TITLE
Colour change in Pokemon blue

### ISSUE THREAD

**Issue by KimiRaiki7** [budude2/openfpga-GBC#52]
In Pokémon Blue the color often changes briefly during battles (if sgb mode is enabled)
Is this a known problem?

**Comment by KimiRaiki7** [comment]
https://github.com/budude2/openfpga-GBC/assets/167031783/0a5a453e-da2e-4cbd-aa0d-05a454746fdc

here you can seethe problem: the graphics change the color during the fight

**Comment by KimiRaiki7** [comment]
> Is it happening in other games? Looks like your pocket is breaking...

No, I only noticed the problem in pokemon blue/red. Did you try it on your pocket?

**Comment by Anatrok** [comment]
I can confirm this behavior.  If you compare to the spiritualized SGB core (or a physical SGB) this brief color cycle does not occur.

I noticed that on booting the game the boot screen is using a green DMG-esque palette before flashing to black, the back to white with the expected SGB palette.

It seems that during any sprite palette swap (Pokémon red/blue does this on SGB so every Pokémon has an accurate palette) this core has a frame where both sprites default back to the dmg-esque palette.  This is just my laymen's guess.

**Comment by budude2** [comment]
That’s odd… Any idea if it does it on mister?

**Comment by Anatrok** [comment]
Looks like this was fixed upstream

https://github.com/MiSTer-devel/Gameboy_MiSTer/commit/05246601b396ad5a55ac6172cb9c29ed08f8d924

**Comment by omernaveedxyz** [comment]
Any chance we could have the fix upstream pulled into this repo?
